tests/FSi/DoctrineExtensions/Tests/Translatable/ListenerTest.php

```diff
@@ -180,7 +206,7 @@ public function testInsertWithTwoTranslationsClearAndLoad()
         $article = $this->_em->find(self::ARTICLE, $article->getId());
 
         $this->assertEquals(
-            3,
+            2,
```

Comment:
This is probably wrong because translatable collection would not be loaded. There are missing tests for persisting/loading translatable collection. `PropertyAccessor` ensured loading such a collection by calling its getter. Now we have to trigger lazy load for collections before setting/getting them